devenv-cli.sh

```diff
@@ -0,0 +1 @@
+source prepare-solvcon-dev.sh
```

Comment:
Instead of using relative path, how about creating a '`ROOT`' path and source related to that?

Reply:
Sounds good. Let's use `WORKING_ROOT`?

Reply:
Would you like it to have a common / SOLVCON-specific prefix?